firmware/config/config.h.template

```diff
@@ -130,4 +130,7 @@
 
 #define MAX_RETRIES 3
 
+// #define MEMORY_DEBUG true
+// #define MEMORY_DEBUG_INTERVAL 5000
```

Comment:
Could/should we combine these?  If MEMORY_DEBUG_INTERVAL is defined then enable+use it, otherwise disable memory_debug?

Reply:
Thanks. Maybe a comment saying that if this value is not defined, then no memory debugging is emitted?